app/server.js

```diff
@@ -15,44 +38,31 @@ const { logger } = require('./logger');
     }
   });
 
-  let intervalId;
-  let app;
-  const pollDuration = 1000;
+  let appType;
+  if (config.loginEnabled === 'false') {
+    appType = 'basic';
+  } else {
+    await isIsapiReady({
+      isapiReady: false, attempt: 1, pollDuration: 1000,
+    });
+    appType = 'auth';
```

Comment:
if you return `'auth'` from the `isIsapiReady` function then you could do something like this:
`const appType = (config.loginEnabled === 'false') ?  'basic' : await isIsapiReady({ attempt: 1, pollDuration: 1000 });`

Reply:
Done similar to what you suggested. I like isIsapiReady returning just a boolean